src/components/TopMenu.tsx

```diff
@@ -44,7 +46,8 @@ function TopMenu(): ReactElement {
   const theme = useTheme()
   const { pathname } = useLocation()
   const activeTab = pathname.split("/")[1] as ActiveTabType
-
+  const { tokenPricesUSD } = useSelector((state: AppState) => state.application)
+  const sdlPrice = tokenPricesUSD?.[SDL_TOKEN.geckoId]
```

Comment:
tokenPrices are keyed by symbol. so you can just use `"SDL"` here

Reply:
I wanted to use what's already there in the constants file.